filecoin-proofs/src/bin/phase2.rs

```diff
@@ -295,18 +244,15 @@ fn blank_winning_post_poseidon_circuit(
         priority: false,
     };
 
-    let public_params = winning_post_public_params(&post_config).unwrap();
+    let public_params = winning_post_public_params::<DefaultOctTree>(&post_config).unwrap();
```

Comment:
Why is this defaulting to a simple oct tree? Doesn't this need to depend on the sector size?